src/main/java/by/onliner/utils/Recorder.java

```diff
@@ -0,0 +1,122 @@
+package by.onliner.utils;
+
+import org.monte.media.Format;
+import org.monte.media.Registry;
+import org.monte.media.math.Rational;
+import org.monte.screenrecorder.ScreenRecorder;
+import ws.schild.jave.*;
+
+import java.awt.*;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static by.onliner.constants.FilePathConstants.SAVE_VIDEO_PATH;
+import static by.onliner.constants.RecorderConstants.*;
+import static org.monte.media.FormatKeys.*;
+import static org.monte.media.VideoFormatKeys.*;
+
+/***
+ * The class is used to record screen and convert video to mp4 format.
+ */
+public class Recorder extends ScreenRecorder {
+
+    private static ScreenRecorder screenRecorder;
+    private static String name;
+
+    /***
+     * Setting video recorder
+     * @param cfg graphics configuration
+     * @param captureArea capture size
+     * @param fileFormat format of the file
+     * @param screenFormat format of the screen
+     * @param mouseFormat format of the mouse
+     * @param audioFormat audio format
+     * @param movieFolder movie file
+     * @param name method name
+     */
+    public Recorder(GraphicsConfiguration cfg, Rectangle captureArea, Format fileFormat,
+                    Format screenFormat, Format mouseFormat, Format audioFormat, File movieFolder, String name)
+            throws IOException, AWTException {
+        super(cfg, captureArea, fileFormat, screenFormat, mouseFormat, audioFormat, movieFolder);
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH.mm.ss");
+        Recorder.name = name + "-" + dateFormat.format(new Date()) + "." + Registry.getInstance().getExtension(fileFormat);
```

Comment:
Why do you reinitialize variable Recorder.name? 
Why do you call variable defined in this class using class name? in what cases do we refer to variables using class name?

Reply:
Fixed